cudax/include/cuda/experimental/__stf/internal/parallel_for_scope.cuh

```diff
@@ -423,7 +452,7 @@ public:
       Fun&& f                  = mv(::std::get<2>(*p));
       const sub_shape_t& shape = ::std::get<3>(*p);
 
-      auto explode_coords = [&](size_t i, deps_t... data) {
+      auto explode_coords = [&](size_t i, typename deps_ops_t::first_type... data) {
```

Comment:
Need a comment to say what that type is (we can't use an alias because it's a pack, not a tuple)